SimTKcommon/include/SimTKcommon/internal/Value.h

```diff
@@ -158,6 +158,21 @@ of the underlying concrete type.
 inline std::ostream& operator<<(std::ostream& o, const AbstractValue& v) 
 {   o << v.toXmlElement(""); return o; }
 
+// TODO document this method.
```

Comment:
better to document this now -- it's unlikely to get revisited later and in any case it would be harder to remember what to say about it then